amundsen_application/static/js/components/SearchPage/SearchFilter/index.tsx

```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
```

Comment:
This feature needs to be completed and this component is not currently used, so I am fine to merge in as is  -- we should be finally able to continue this work next week any way.